.github/workflows/e2e.yml

```diff
@@ -207,8 +232,10 @@ jobs:
 
   e2e:
     name: End to end tests
-    runs-on: ubuntu-latest
-    needs: install-deps
+    runs-on: windows-latest
+    needs:
+      - install-deps
+      - install-deps-windows
```

Comment:
Needs the matrix output from `install-deps` and the cache from `install-deps-windows`.